components/kyma-environment-broker/internal/broker/plans.go

```diff
@@ -172,8 +172,8 @@ func empty() *map[string]interface{} {
 	return &empty
 }
 
-func createSchema(machineTypes, regions []string, additionalParams, update bool) *map[string]interface{} {
-	properties := NewProvisioningProperties(machineTypes, regions, update)
+func createSchema(machineTypesDisplay map[string]string, machineTypes, regions []string, additionalParams, update bool) *map[string]interface{} {
```

Comment:
note that even though `machineTypesDisplay` contains all of `machineTypes`, because KEB should maintain predefined order of `machineType` values, it's best to keep passing both, the properly ordered slice as well as unordered map with display info.